_data/team.yml

```diff
@@ -235,7 +235,7 @@
   fullDescription: >
     <p>Software craftsman who has worked in Spain (Madrid and Valencia) and now is based in London. Always interested in improving his skills.</p>
     <p>Christian started his career as a backend developer but some years later gave a switch and started developing mobile applications. After some years, he decided to go back to the backend world to continue learning new technologies and to work on more complex domains. Currently he is interested in test driven development, domain driven design and he is learning functional programming as his second programming paradigm.</p>
```

Comment:
Sorry, not sure what is meant by 'gave a switch' 
Do you mean 'switched to mobile application development'